privacyidea/lib/tokens/hotptoken.py

```diff
@@ -262,7 +317,12 @@ def update(self, param, reset_failcount=True):
         else:
             self.set_otplen(get_from_config("DefaultOtpLen", 6))
 
-
+        # Add twostep settings to the tokeninfo
+        if twostep_init:
```

Comment:
Can we move this to the top (line 283), so that we only have *one* ``if two_step_init`` in the update method?

      if twotstep_init:
            # do all the tokeninfo 2step)
            if  "2step_serversize" in upd_param:
                 # do the stuff in line 286

Reply:
815577b943a9090ec4211bfac7bf8655130c3a31 does this.